content/influxdb/v3/reference/sql/operators/other.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Other SQL operators
+list_title: Other operators
+description: >
+  SQL supports other miscellaneous operators that perform various operations.
+menu:
+  influxdb_v3:
+    name: Other operators
+    parent: Operators
+weight: 305
+list_code_example: |
+  |    Operator    | Meaning                  | Example                                 | Result        |
+  | :------------: | :----------------------- | :-------------------------------------- | :------------ |
+  |     `\|\|`     | Concatenate strings      | `'Hello' \|\| ' world'`                 | `Hello world` |
```

Comment:
Could this be in a separate "Strings operators" page?

Reply:
Possibly in the future. The reason I didn't now is because we'd have two pages with single operators.